section-09-differential-testing/model-package/differential_tests/test_model_accuracy.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+import numpy as np
+from regression_model.predict import make_prediction
+
+
+def test_make_prediction(sample_input_data, persist_results):
+    # Given
+    expected_first_prediction_value = 113422
```

Comment:
why do we hard code this value if we don't test it?

Also, we are not testing accuracy or performance here. I am confused. What is the aim of this file?